web/packages/teleport/src/services/userEvent/types.ts

```diff
@@ -98,6 +98,7 @@ export enum DiscoverEvent {
   PrincipalsConfigure = 'tp.ui.discover.principals.configure',
   TestConnection = 'tp.ui.discover.testConnection',
   Completed = 'tp.ui.discover.completed',
+  Overview = 'tp.ui.discover.overview',
```

Comment:
just fyi, this requires backend changes, i have to create another event anyways (github), so i'll just add this as well

Reply:
Thanks!